src/internal/images/pull.go

```diff
@@ -45,7 +46,7 @@ func PullAll(buildImageList []string, imageTarballPath string) map[name.Tag]v1.I
 		if err != nil {
 			spinner.Fatalf(err, "Unable to pull the image %s", src)
 		}
-		imageCachePath := config.GetImageCachePath()
+		imageCachePath := filepath.Join(config.GetCachePath(), "images")
```

Comment:
Can we turn the "images" string into a const somewhere? It would be nice to track what subdirectories Zarf is utilizing within its cache.